concepticondata/conceptlists/Stoll-1884-259.tsv

```diff
@@ -0,0 +1,260 @@
+Stoll-1884-259-47	47	Knie	knee	1371	KNEE
+Stoll-1884-259-48	48	Fuss	foot	1301	FOOT
+Stoll-1884-259-49	49	Knochen	bone	1394	BONE
+Stoll-1884-259-50	50	Stützpfeiler	column	2045	COLUMN
+Stoll-1884-259-51	51	Wand	wall	933	WALL (OF HOUSE)
+Stoll-1884-259-52	52	Bett	bed	1663	BED
+Stoll-1884-259-53	53	Bastmatte	bast mat	195	MAT
+Stoll-1884-259-54	54	Wolltuch	wool cloth		
+Stoll-1884-259-55	55	Blut	blood	946	BLOOD
+Stoll-1884-259-56	56	Harn	urine	1142	URINE
+Stoll-1884-259-57	57	Foeces	faeces	675	FAECES (EXCREMENT)
+Stoll-1884-259-58	58	Schweiss	sweat	125	SWEAT (SUBSTANCE)
+Stoll-1884-259-59	59	Dorf	village	930	VILLAGE
+Stoll-1884-259-60	60	Haus	house	1252	HOUSE
+Stoll-1884-259-61	61	Löffel	spoon	1378	SPOON
+Stoll-1884-259-62	62	Guacal	hollowed-out gourd for drinking		
+Stoll-1884-259-63	63	Jicara	gourd dipper		
+Stoll-1884-259-64	64	Wasserkrug	water pitcher	3476	WATER PITCHER
+Stoll-1884-259-65	65	Kürbisflasche	gourd-shaped bottle		
+Stoll-1884-259-66	66	Korb	basket	1539	BASKET
+Stoll-1884-259-67	67	Stirnband	headband	330	HEADBAND
+Stoll-1884-259-68	68	Blasrohr	blowgun	432	BLOWGUN
+Stoll-1884-259-69	69	Bogen	bow	994	BOW
+Stoll-1884-259-70	70	Pfeil	arrow	977	ARROW
+Stoll-1884-259-71	71	Agavegarn	agave fiber		
+Stoll-1884-259-72	72	Schnur	string	362	STRING
+Stoll-1884-259-73	73	Tragnetz	netbag	293	NETBAG
+Stoll-1884-259-74	74	Tuch	cloth	1616	CLOTH
+Stoll-1884-259-75	75	Markt	market	633	MARKET
+Stoll-1884-259-76	76	Hut	hat	771	HAT
+Stoll-1884-259-77	77	Sandalen	sandals		
+Stoll-1884-259-78	78	Kamm	comb	1475	COMB
+Stoll-1884-259-79	79	Beinkleid	trousers	809	TROUSERS
+Stoll-1884-259-80	80	Weiberrock	skirt	1600	SKIRT
+Stoll-1884-259-81	81	Weiberhemd	shirt worn by women		
+Stoll-1884-259-82	82	Wasser	water	948	WATER
+Stoll-1884-259-83	83	Salz	salt	1274	SALT
+Stoll-1884-259-84	84	Atole	atole		
+Stoll-1884-259-85	85	Tortilla	tortilla		
+Stoll-1884-259-86	86	Fleisch	meat	634	MEAT
+Stoll-1884-259-87	87	Bohnen	beans	832	BEAN
+Stoll-1884-259-88	88	Maiskolben	corn on the cob		
+Stoll-1884-259-89	89	Maiskörner	corn kernels		
+Stoll-1884-259-90	90	Unreifer mais	unripe corn		
+Stoll-1884-259-91	91	Olote	corncob		
+Stoll-1884-259-92	92	Maisfeld	corn field	449	CORN FIELD
+Stoll-1884-259-93	93	Ei	egg	744	EGG
+Stoll-1884-259-94	94	Vogel	bird	937	BIRD
+Stoll-1884-259-95	95	Truthahn	turkey	2386	TURKEY
+Stoll-1884-259-96	96	Zopilote	vulture	1185	VULTURE
+Stoll-1884-259-97	97	Eufe	-		
+Stoll-1884-259-98	98	Schwein	pig	1337	PIG
+Stoll-1884-259-99	99	Hund	dog	2009	DOG
+Stoll-1884-259-100	100	Katze	cat	1208	CAT
+Stoll-1884-259-101	101	Hase	hare	1190	HARE
+Stoll-1884-259-102	102	Hahn	rooster	1511	ROOSTER
+Stoll-1884-259-103	103	Henne	hen	1514	HEN
+Stoll-1884-259-104	104	Vogelnest	bird's nest	539	NEST
+Stoll-1884-259-105	105	Feder	feather	1201	FEATHER
+Stoll-1884-259-106	106	Flügel	wing	1257	WING
+Stoll-1884-259-107	107	Schwanz	tail	1220	TAIL
+Stoll-1884-259-108	108	Jaguar	jaguar	1250	JAGUAR
+Stoll-1884-259-109	109	Cuguar	cougar		
+Stoll-1884-259-110	110	Reh	roe deer		
```

Comment:
1936?

Reply:
1936 is a cover term for a variety of deer species, and the roe deer is just one of them.

Reply:
Gotcha!